tests/suites/test_suite_pkwrite.data

```diff
@@ -7,12 +7,12 @@ depends_on:MBEDTLS_RSA_C
 pk_write_pubkey_check:"../framework/data_files/server1.pubkey.der":TEST_DER
 
 Public key write check RSA 4096
-depends_on:MBEDTLS_RSA_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_PEM_WRITE_C
-pk_write_pubkey_check:"../framework/data_files/rsa4096_pub.pem":TEST_PEM
+depends_on:MBEDTLS_RSA_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_PEM_WRITE_C:MBEDTLS_TEST_PK_ALLOW_RSA_KEY_PAIR_4096
+pk_write_pubkey_check:"../../framework/data_files/rsa4096_pub.pem":TEST_PEM
```

Comment:
How come this line needs `../../framework` but line 7 only needs `../framework`?

Reply:
Ops, sorry, that's one of the 2 files who had conflicts during the rebase and apprently I missed a change here :/